content/en/platform/corda/4.8/open-source/key-concepts-node.md

```diff
@@ -20,40 +20,40 @@ title: Nodes
 
 ## Summary
 
-* *A node is JVM run-time with a unique network identity running the Corda software*
-* *The node has two interfaces with the outside world:*
-  * *A network layer, for interacting with other nodes*
-  * *RPC, for interacting with the node’s owner*
-* *The node’s functionality is extended by installing CorDapps in the plugin registry*
+* Nodes represent individuals and businesses on a Corda network.
+* A node runs within a [Java Virtual Machine (JVM)](https://www.infoworld.com/article/3272244/what-is-the-jvm-introducing-the-java-virtual-machine.html) and has a unique network identity running the Corda software.
```

Comment:
This doesn't quite make sense. A node must have a unique identity, which corresponds to a legal identity. This is an identity, it's not what 'runs the Corda software.' Can you look at this again and split those two concepts out a bit?